lib/net/imap/response_parser.rb

```diff
@@ -1321,6 +1341,23 @@ def number
         return token.value.to_i
       end
 
+      def set
```

Comment:
Can we rename this `uid_set` to more closely match the RFC4315 and RFC9051 names?  In addition to simply sticking as closely to the RFC grammar and names as we can, `sequence-set` and `uid_set` do have slightly different definitions.

```suggestion
      # RFC-4315 (UIDPLUS) or RFC9051 (IMAP4rev2):
      #      uid-set         = (uniqueid / uid-range) *("," uid-set)
      #      uid-range       = (uniqueid ":" uniqueid)
      #                          ; two uniqueid values and all values
      #                          ; between these two regardless of order.
      #                          ; Example: 2:4 and 4:2 are equivalent.
      #      uniqueid        = nz-number
      #                          ; Strictly ascending
      def uid_set
```

Reply:
Ah yes that makes more sense :)